schemas/core/product-option.json

```diff
@@ -7,6 +7,10 @@
     { "$ref": "#/definitions/withCost" }
   ],
   "definitions": {
+    "ref": {
```

Comment:
may be good to have description to not be confused with $ref

Reply:
sound good but I have no idea what it is used for

Reply:
"A number to reference to leg productOption. leg.productOption === productOption.ref"